src/index.ts

```diff
@@ -16,10 +16,29 @@ export class TypeScriptPlugin {
   serverless: Serverless.Instance
   options: Serverless.Options
   hooks: { [key: string]: Function }
+  utils: Serverless.Utils
+  commands: Serverless.CommandsDefinition
+  watchProgress?: Serverless.Progress
 
-  constructor(serverless: Serverless.Instance, options: Serverless.Options) {
+  constructor(serverless: Serverless.Instance, options: Serverless.Options, utils) {
     this.serverless = serverless
     this.options = options
+    this.utils = utils
+
+    this.commands = {
```

Comment:
This ideally should be propsed with another PR which can also land in current major of a plugin

Reply:
👍  I've opened https://github.com/serverless/serverless-plugin-typescript/pull/257